### ISSUE TITLE
TypeError : Argument 4 passed to PhpOffice\PhpSpreadsheet\Writer\Xlsx\Worksheet::writeAttributeIf() must be of the type string

### ISSUE THREAD

**Issue by laurin1** [PHPOffice/PhpSpreadsheet#1505]
I just updated to 1.13.0 and now when I try to save an XLS file that I just imported, I get this error:

TypeError : Argument 4 passed to PhpOffice\PhpSpreadsheet\Writer\Xlsx\Worksheet::writeAttributeIf() must be of the type string, null given, called in C:\inetpub\Intranet_Local\vendor\phpoffice\phpspreadsheet\src\PhpSpreadsheet\Writer\Xlsx\Worksheet.php on line 1164
C:\inetpub\Intranet_Local\vendor\phpoffice\phpspreadsheet\src\PhpSpreadsheet\Writer\Xlsx\Worksheet.php:453
C:\inetpub\Intranet_Local\vendor\phpoffice\phpspreadsheet\src\PhpSpreadsheet\Writer\Xlsx\Worksheet.php:1164
C:\inetpub\Intranet_Local\vendor\phpoffice\phpspreadsheet\src\PhpSpreadsheet\Writer\Xlsx\Worksheet.php:1040
C:\inetpub\Intranet_Local\vendor\phpoffice\phpspreadsheet\src\PhpSpreadsheet\Writer\Xlsx\Worksheet.php:67
C:\inetpub\Intranet_Local\vendor\phpoffice\phpspreadsheet\src\PhpSpreadsheet\Writer\Xlsx.php:267
 C:\inetpub\Intranet_Local\library\Intranet\PhpOffice\XlsxWriter.php:20
 C:\inetpub\Intranet_Local\phpunit\library\Intranet\PHPExcelTest.php:52

I am not making any changes to the file, just import, then save:

```php
$sFileImportedFromEmail      = _TEST_DATA_PATH."/imported_from_email01.xls";
$sFileImportedFromEmailSaved = _TEST_DATA_PATH."/imported_from_email_saved01.xls";

$this->expectOutputString(
	"");

try{

	$oSpreadsheet =
		IOFactory::load(
			$sFileImportedFromEmail);
	$oXlsxWriter  =
		new XlsxWriter(
			$oSpreadsheet);

	$oXlsxWriter->save(
		$sFileImportedFromEmailSaved);

}
catch(Exception $oException){

	Exceptions::convertToError(
		$oException);

	$aSpreadsheet = [];

}
```

Here is the file that I am importing:

https://drive.google.com/file/d/1xrbe_knlKBeqAMpmNGKUYFMatP_UPteX/view?usp=sharing

**Comment by jujuo0o** [comment]
Getting the exact same error while opening an xlsx template with cells having "FILL COLOR"

**Comment by Sediug** [comment]
I'm getting the same problem loading an XLSX which contains multiple sheets and trying to create a file foreach sheet. Using version 1.18.0

```php
public function extractSheetsToFiles(callable $outputWriter): void
    {
        $spreadsheet = IOFactory::load($this->getLocalFilePath());

        foreach ($spreadsheet->getAllSheets() as $sheet) {
            if ($this->isSheetEmpty($sheet)) {
                continue;
            }

            $newSpreadsheet = new Spreadsheet();
            $newSpreadsheet->removeSheetByIndex(0);
            $newSpreadsheet->addSheet($sheet->copy(), 0);
            $newSpreadsheet->setActiveSheetIndex(0);

            $this->sanitizeSheetProperties($newSpreadsheet->getActiveSheet());

            $memoryStream = fopen('php://memory', 'rw+');
            $xlsxWritter = new Xlsx($newSpreadsheet);
            $xlsxWritter->save($memoryStream);

            rewind($memoryStream);

            $outputWriter($sheet->getTitle(), $memoryStream);

            fclose($memoryStream);
        }
    }
```

**Comment by oleibman** [comment]
Fixed by PR #1525 in June 2020.
